base_example/configure_tower_export_model.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Playbook to configure ansible tower post installation
+  hosts: localhost
+  connection: local
+  vars:
+    workflow_job_templates_secure_logging: false
+  # Define following vars here, or in tower_configs/tower_auth.yml
+  # tower_hostname: ansible-tower-web-svc-test-project.example.com
+  # tower_username: admin
+  # tower_password: changeme
+
+  collections:
+   - awx.awx
+   - sean_m_sullivan.tower_configuration
```

Comment:
Needs changing to redhat_cop.tower_configuration

Reply:
See above, Temporary.

Reply:
Removed this file